### DISCUSSION TITLE
Livesplit Overlay integration

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#2919] [General]
This was broken when the switch from GLFW to SDL happened if anyone wants to pick this up.

**Comment by xTVaser** [comment]
Can you provide some detail on whatever this overlay integration is?  Is it an OBS thing? I see nothing in livesplit that mentions an overlay

**Reply by Zedb0T** [reply]
Yeah this was suggested by someone who isnt me so I am not super familiar with what this could be... I tried looking it up and yeah I cant find anything conclusive either.

**Reply by Zedb0T** [reply]
https://discord.com/channels/756287461377703987/1137250075056283778/1137250075056283778
